main.js

```diff
@@ -1,79 +1,74 @@
-const electron = require('electron');
-const app = electron.app;  // Module to control application life.
-const BrowserWindow = electron.BrowserWindow;  // Module to create native browser window.
-const path = require('path');
+const electron = require('electron')
+const app = electron.app  // Module to control application life.
+const BrowserWindow = electron.BrowserWindow  // Module to create native browser window.
+const path = require('path')
 
-var mainWindow = null;
+var mainWindow = null
 
 // Quit when all windows are closed.
 
-const onQuit = function() {
+const onQuit = function () {
   app.quit()
-};
+}
 
 const encodeArgsAsQueryString =
   args =>
-  args.reduce
-  ( ({flag, query}, arg) =>
-    ( /^--\w+$/.test(arg) // If new flag swap flag.
-    ? { flag: arg.substr(2)
-      , query:
-        ( query === ""
+  args.reduce(({flag, query}, arg) =>
+    (/^--\w+$/.test(arg) // If new flag swap flag.
+    ? { flag: arg.substr(2),
+       query:
+        (query === ''
         ? `${arg.substr(2)}`
         : `${query}&${arg.substr(2)}`
         )
       }
     : flag == null  // If there is no flag then skip.
-    ? { flag
-      , query
+    ? { flag,
+       query
```

Comment:
I didn't see any indentation "errors" caught by the linter, so I don't think it's even checking it. I'll fix those indentations issues with pleasure ;)